inst/tutorials/airline-safety/case_study1.Rmd

```diff
@@ -37,14 +37,14 @@ This case study is based on an [article](https://fivethirtyeight.com/features/sh
 # data$period = as.factor(ifelse(data$period == "85_99", "1985_1999", "2000_2014"))
 # 
 # write.table(data,
-#            file = "~/Documents/TA_Econometrics/Repo_github/ScPoEconometrics/airline-safety.csv",
+#            file = "~/Documents/TA_Econometrics/Repo_github/ScPoEconometrics/inst/datasets/airline-safety.csv",
 #            row.names = F,
 #            sep = ",")
 ```
 
 To load the dataset, you can run the following command (NB : it uses the `readr` package)
 ```{r include=TRUE, cache.comments=FALSE}
-# load data from our github repo 
+# load data from our github repo
 data=read_csv("https://raw.github.com/pvilledieu/ScPoEconometrics/airline_safety/inst/datasets/airline-safety.csv")
```

Comment:
this will be the wrong url. 
you could also say to do `read.csv(system.file(package = "ScPoEconometrics", "datasets", "airline-safety.csv"))` (no need for readr...)

Reply:
yes it was to check if it was working before you merge into your repo 
ok perfect I change for your command !